### PR TITLE
Extra documentation for networking

### PR DESCRIPTION
**Pull request by RemyDeWolf**
Everything is consolidated under **Features/Networking**

- Network Overview (existing and new content)
- High availability (new)
- Network Monitoring (added viz here)
- Subnetting (new)
- Using Other Service Meshes (moved in its own page)

preview https://deploy-preview-165--opta-docs.netlify.app/features/networking/

### CODE REVIEW

File: content/en/Features/networking/high_availability.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+title: "High availability"
+linkTitle: "High availability"
+date: 2022-03-11
+weight: 2
+draft: false
+description: >
+  High availability to provide resiliency in case of zone outages
+---
+
+## Kubernetes Ingress - high availability
```
Comment (NitinAgg) [inline content/en/Features/networking/high_availability.md:11]: There is also linkerd high availability field

File: content/en/Features/networking/network_monitoring.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+draft: false
+description: >
+  How to monitor the network
+---
+
+### Cloud provider network networking
+
+Each cloud provider has some preconfigured service to monitor networking such as [Amazon CloudWatch](https://aws.amazon.com/cloudwatch/), [Google operations suite](https://cloud.google.com/products/operations) and [Azure Monitor](https://azure.microsoft.com/en-us/services/monitor/).
+
+### Linkerd Viz
```
Comment (NitinAgg) [inline content/en/Features/networking/network_monitoring.md:15]: We have a tutorial for this somewhere as well. Let's make sure we don't duplicate this

**Review by NitinAgg — COMMENTED** [review]
Love this! This should definitely be a big blog post

File: content/en/Features/networking/network_monitoring.md (hunk 2 above)
Comment (RemyDeWolf) [reply content/en/Features/networking/network_monitoring.md:15]: github doesn't show the diff on this but this section is moved from `content/en/Features/service2service.md`  (it shows as deleted in the PR)

File: content/en/Features/networking/high_availability.md (hunk 1 above)
Comment (RemyDeWolf) [reply content/en/Features/networking/high_availability.md]: oh gotcha, I missed that one, I updated this section and the diagrams to show this.

File: content/en/Features/networking/subnetting.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+- VPC 10.0.0.0/16, 65,536 but it comes to 12,288 usable IPs because of availability zone restrictions.
+- For public subnets AZs, 10.0.0.0/21 2,048 IPs, 10.0.8.0/21 2,048 IPs, 10.0.16.0/21 2,048 IPs. They are used only with public facing resources such as NAT Gateway, Load Balancers.
+- For private subnets AZs, 10.0.128.0/21 2,048 IPs, 10.0.136.0/2 2,048 IPs, 10.0.144.0/21 2,048 IPs. They are used for everything else, ex: kubernetes workloads.
+
+### GCP
+
+- VMs 10.0.0.0/19 - 8,192 IPs
+- k8s pods 10.0.32.0/19  - 8,192 IPs
+- k8s services 10.0.64.0/20 - 4,096 IPs
+- k8s master - 10.0.80.0/28 - 13 IPs (number of clusters - AWS limitation)
```
Comment (juandiegopalomino) [inline content/en/Features/networking/subnetting.md:34]: GCP limitation lol

**Pull request merged**